apps/explorer/src/pages/home/Home.tsx

```diff
@@ -93,10 +93,10 @@ function Home() {
                     </TabPanels>
                 </TabGroup>
             </div>
-            {/* TODO: Add the popular packages component here :) */}
-            {/* <div style={{ gridArea: 'packages' }} className="mt-5 bg-gray-60">
-                Popular packages
-            </div> */}
+
```

Comment:
could we make sure this looks good on all breakpoints for the new design?

Reply:
double checked, looks good